### PR TITLE
Quickfacts update revised 02/01/2019 (Russian)

### PR DESCRIPTION
**Pull request by v1docq47**
Edit:

*  outreach-docs/monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md

Calling to review:
@monerorus @LvMsterfild @TheFuzzStone

### CODE REVIEW

**Comment by erciccione** [comment]
@lh1008 could you review this and check if the data here is correct?

File: monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md [hunk 1]
```diff
@@ -57,46 +57,46 @@ Monero — это слово на языке эсперанто. Создате
...
++ **Количество транзакций в час:** 149 (Средний показатель за 30 дней)
 
-+ **Ядра CPU, обеспечивающие безопасность сети:** 14,499,178
++ **Ядра CPU, обеспечивающие безопасность сети:** 22,234,297
 
-+ **Monero в обращении:** 16,570,126 XMR (Приблизительно)
++ **Monero в обращении:** 16,758,217 XMR (Приблизительно)
 
-+ **Рыночная капитализация:** $1,503,928,296 USD (~0.77% от общего объема капитализации)
++ **Рыночная капитализация:** $743,944,945 USD (~0.77% от общего объема капитализации)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md]: We found the information we had posted was incorrect and thanks to @el00ruobuob we were able to fix it. This is the string with the correct information.

**Market Capitalization:** $732,932,071 USD (~0.65% of total cryptocurrency market cap)

File: monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ _При "дополнительной эмиссии" (0,6 XMR/блок) к 2040
...
 
-**Аппаратные кошельки -** Trezor Model T hardware wallet added Monero to their **[supported coins](https://trezor.io/coins/#XMR)**. **[Ledger hardware wallet](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** already available through official Monero GUI wallet. The Monero community funded a team that is building **[Kastelo, an open source hardware wallet](http://kastelo.org/)**, built on the Monero ethos. Both of these hardware wallet options have full Monero support. 
+**Аппаратные кошельки -** популярные аппаратные кошельки Ledger и Trezor работают над добавлением Monero. (В Ledger уже реализована поддержка Monero начиная с версии CLI 0.12.2 и GUI 0.12.3) Сообщество Monero финансирует команду, которая **[занимается разработкой унифицированного аппаратного кошелька](http://kastelo.org/)** для Monero. Предполагается, что оба этих аппаратных кошелька будут введены в опытную эксплуатацию к концу 2018 года.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md]: We also made an addition to the last sentence of this paragraph. Added some missing words and made it a little more understandable. This is the last sentence: 

Both Ledger and Trezor hardware wallet options were anticipated to be operational by the end of 2018.

You can take a look at #99 and see the change in the last sentence of the paragraph.

File: monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md [hunk 3]
```diff
@@ -57,46 +57,46 @@ Monero — это слово на языке эсперанто. Создате
 **-er-:** малая часть чего-либо
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md]: We need to leave spaces between sentences in order to separate the lines. Let me show you:

**mon-:** деньги

**-er-:** малая часть чего-либо

**-o :** вещь (выражение - существительное)

File: monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md (hunk 1 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md]: fix

File: monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md (hunk 2 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md]: fix

File: monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md (hunk 3 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/quickfacts/quickfacts_ru.md]: fix

**Comment by v1docq47** [comment]
@lh1008 thank you for the indicated inaccuracies. corrected all comments ;)

**Pull request merged**